docs/migration/migration1120to1130.adoc

```diff
@@ -131,3 +131,12 @@ A változtatások nem eredményeznek átállási munkálatokat, visszafelé komp
 ==== Átállás
 
 A változtatások nem eredményeznek átállási munkálatokat, visszafelé kompatibilis.
+
+=== coffee-rest
+
+* Deltaspike álltal nyújtott ProjectStage leváltásra került egy coffee-ba implementált egyszerűsített változatra. ProjectStage.Production megmaradt!
```

Comment:
A projectStage kezeleserol a sima coffee-rest bekezdesben is irni kell valamit. Ugy mintha a deltaspike valtozas sosem nem is letezett volna. Tehat mire szolgal hogy kell hasznalni, mi a konfiguracioja...

Reply:
elkészült